### DISCUSSION TITLE
Disable default sort on third click?  (react-table v7)

### DISCUSSION THREAD

**Discussion by msmith1114** [TanStack/table#2235] [General]
Probably a confusing title, but I am just now getting used to react-table v7. I have enabled sorting (using the examples) however I can't quite figure out a way to disable it going back to "default" sort after the third click (typically it'll go DESC-->ASC-->DEFAULT(no sort)). Is there a way to disable it going back to default on the third click of the column?

Thanks!

**Comment by pszafer** [comment]
I think you want set to true `disableSortRemove` option.
